platform/view/services/storage/vault/vaultstore_test.go

```diff
@@ -26,6 +30,133 @@ const (
 	unknown
 )
 
+type matrixItem struct {
+	pagination driver.Pagination
+	matcher    []types.GomegaMatcher
+}
+
+var matrix = []matrixItem{
+	{
+		pagination: common.NewNoPagination(),
+		matcher: []types.GomegaMatcher{
+			gomega.ConsistOf(
+				gomega.HaveField("TxID", gomega.Equal("txid1")),
```

Comment:
Only for the `gomega` package use dot imports, because it makes test more fluent.